src/agent0/ethpy/hyperdrive/deploy.py

```diff
@@ -230,12 +235,32 @@ def _deploy_hyperdrive_factory(
         account=deploy_account_addr,
         constructorArgs=HyperdriveFactoryContract.ConstructorArgs(factory_deploy_config, "HyperdriveFactory"),
     )
+
+    lp_math_contract = LPMathContract.deploy(w3=web3, account=deploy_account_addr)
```

Comment:
I'm not sure of all the uses cases for this deploy method, but should we allowing passing in an existing lp_math_contract_address or some kinda external libraries obj?

Reply:
Deploy only gets called when we deploy on a new chain. Agent0 doesn't support deploying on an existing chain, so no need to handle the case when this contract already exists.